index.html

```diff
@@ -122,7 +122,7 @@ <h2>
         </div>
 
         
-        <div class="project_tag"><button>See project</button></div>
+        <div class="project_tag"><button class="see-more2">See project</button></div>
```

Comment:
- [ ] Please kindly ensure that the markup for the projects section is entirely removed and created dynamically using Javascript